sources/packages/backend/apps/db-migrations/src/sql/Institution/Add-col-other-regulating-body.sql

```diff
@@ -0,0 +1,4 @@
+-- Add other_regulating_body column to institutions table
+ALTER TABLE sims.institutions ADD COLUMN IF NOT EXISTS other_regulating_body VARCHAR(100);
+
+COMMENT ON COLUMN sims.institutions.other_regulating_body IS 'Allows the user to enter the description if the option other is selected from the regulating body dropdown';
```

Comment:
The comment can avoid client specific content like `Allows the user to enter the description` and just convey the business context here. (e.g. Other institution regulating body)

Reply:
Thanks Dheepak. Will make the change.